diff --git a/bin/pmd b/bin/pmd
index 73476fd..ee05070 100755
--- a/bin/pmd
+++ b/bin/pmd
@@ -45,51 +45,12 @@ java_heapsize_settings() {
     esac
 }
 
-
-set_lib_dir() {
-  if [ -z "${LIB_DIR}" ]; then
-    # Allow for symlinks to this script
-    if [ -L "$0" ]; then
-      local script_real_loc=$(readlink "$0")
-    else
-      local script_real_loc=${BASH_SOURCE[0]:-${(%):-%x}}
-    fi
-    local script_dir=$(dirname "${script_real_loc}")
-
-    pushd "${script_dir}/../lib" >/dev/null
-    readonly LIB_DIR=$(pwd -P)
-    popd >/dev/null
-  fi
-}
-
 check_lib_dir() {
   if [ ! -e "${LIB_DIR}" ]; then
     echo "The jar directory [${LIB_DIR}] does not exist"
   fi
 }
 
-set_conf_dir() {
-  if [ -z ${CONF_DIR} ]; then
-    # Allow for symlinks to this script
-    if [ -L $0 ]; then
-      local script_real_loc=$(readlink "$0")
-    else
-      local script_real_loc=${BASH_SOURCE[0]:-${(%):-%x}}
-    fi
-    local script_dir=$(dirname "${script_real_loc}")
-
-    pushd "${script_dir}/../conf" >/dev/null
-    readonly CONF_DIR=$(pwd -P)
-    popd >/dev/null
-  fi
-}
-
-check_conf_dir() {
-  if [ ! -e "${CONF_DIR}" ]; then
-    echo "The configuration directory [${CONF_DIR}] does not exist"
-  fi
-}
-
 function script_exit() {
     echo "$1" >&2
     exit 1
@@ -155,9 +116,9 @@ jre_specific_vm_options() {
 
 function add_pmd_classpath() {
     if [ -n "$classpath" ]; then
-        classpath="$classpath:${CONF_DIR}:${LIB_DIR}/*"
+        classpath="$classpath:${LIB_DIR}/*"
     else
-        classpath="${CONF_DIR}:${LIB_DIR}/*"
+        classpath="${LIB_DIR}/*"
     fi
 }
 
@@ -192,10 +153,7 @@ is_cygwin
 
 check_java
 
-set_lib_dir
 check_lib_dir
-set_conf_dir
-check_conf_dir
 
 convert_cygwin_vars
 
